### ISSUE TITLE
Error and disconnect from server when try to create space station when idDimensionOverworld has nonzero value

### ISSUE THREAD

**Issue by Krysinski** [micdoodle8/Galacticraft#2242]
Cant create space station by pressing button: it return me to Celestial Map without any selected object (like when i click on space). In SP i can create station by click on resource list (maybe inventory visual bug). In MP i get error and disconnect from server.
FML log: http://pastebin.com/2bfi3QxC
Server console:

```
[23:29:23 ERROR]: There was a critical exception handling a packet on channel GalacticraftCore
java.lang.NullPointerException
        at micdoodle8.mods.galacticraft.core.network.PacketSimple.handleServerSide(PacketSimple.java:956) ~[PacketSimple.class:?]
        at micdoodle8.mods.galacticraft.core.network.GalacticraftPacketHandler.channelRead0(GalacticraftPacketHandler.java:27) ~[GalacticraftPacketHandler.class:?]
        at micdoodle8.mods.galacticraft.core.network.GalacticraftPacketHandler.channelRead0(GalacticraftPacketHandler.java:12) ~[GalacticraftPacketHandler.class:?]
        at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98) ~[SimpleChannelInboundHandler.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[MessageToMessageDecoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.embedded.EmbeddedChannel.writeInbound(EmbeddedChannel.java:169) ~[EmbeddedChannel.class:?]
        at cpw.mods.fml.common.network.internal.FMLProxyPacket.func_148833_a(FMLProxyPacket.java:77) [FMLProxyPacket.class:1.7.10-1614.57]
        at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:245) [ej.class:?]
        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:181) [nc.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:1023) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:432) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:841) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:693) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
```

Server:
Thermos-1.7.10-1614-57 (also tested on 46 and 55)
Forge 1614
mods:
AdvancedSolarPanel-1.7.10-3.5.1.jar
appliedenergistics2-rv3-beta-6-client.jar
BiblioCraft[v1.11.4][MC1.7.10].jar
BiblioWoods[Forestry][v1.7].jar
catwalks-2.0.4.jar
Chisel2-2.5.1.44.jar
[CL]DynamicLights-1.7.10.jar
[CL]fastcraft-1.21.jar
[CL]IC2 Backpack HUD v3.0 - MC 1.7.10.zip
[CL]InventoryTweaks-1.59-dev-152.jar
[CL]MemoryCleaner-1.4-mc.1.7.10.jar
[CL]mod_chatBubbles.litemod
[CL]mod_voxelMap_1.5.19_for_1.7.10.litemod
[CL]mod_worldeditcui_1.7.10_00_mc1.7.10.litemod
[CL]MouseTweaks-2.4.4-mc1.7.10.jar
[CL]neiaddons-1.12.14.40-mc1.7.10.jar
[CL]NEIIntegration-MC1.7.10-1.1.1.jar
[CL]OptiFine_1.7.10_HD_U_C1.jar
[CL]OptifineRus-1.0.jar
[CL]StatusEffectHUD-client-1.27.jar
[CL]Waila-1.5.10_1.7.10.jar
[CL]WAILAPlugins-MC1.7.10-0.2.0-23.jar
CodeChickenCore-1.7.10-1.0.7.47-universal.jar
coroutil-1.7.10-1.1.5.jar
CustomOreGen-1.7.10-1.2.24.jar
CyanosLootableBodies-1.7.10-backport_1.3.7.jar
EnderCore-1.7.10-0.2.0.34_beta.jar
EnderIO-1.7.10-2.3.0.425_beta.jar
[FIX-CL]CoFHCore-[1.7.10]3.1.2-325.jar
forestry_1.7.10-4.2.14.62.jar
GalacticraftCore-1.7-3.0.12.460.jar
Galacticraft-Planets-1.7-3.0.12.460.jar
GraviSuite-1.7.10-2.0.3.jar
HardcoreDarkness-MC1.7.10-1.7.jar
ic2ca-1.7.10-1.14.5.06.jar
IC2NuclearControl-2.4.1a.jar
immibis-core-59.1.4.jar
industrialcraft-2-2.2.822-experimental-client.jar
ironchest-1.7.10-6.0.62.742-universal.jar
MicdoodleCore-1.7-3.0.12.460.jar
MineTweaker3-1.7.10-3.0.10B.jar
NotEnoughItems-1.7.10-1.0.5.120-universal.jar
redlogic-59.1.13.jar
ThermalDynamics-[1.7.10]1.1.0-161client.jar
ThermalExpansion-[1.7.10]4.1.2-240.jar
ThermalFoundation-[1.7.10]1.2.3-112.jar
Treecapitator-universal-2.0.4.jar
weaponmod-1.14.3.jar
zombieawareness-1.9.6 for MC v1.7.10.jar
Ztones-1.7.10-2.2.1.jar

Plugins: Multiverse-Core-2.5 and other...

**Comment by Krysinski** [comment]
I can create Space station wihtout errors if set this to 0 in GC config:
# Dimension ID for the Overworld (as seen in the Celestial Map)
I:idDimensionOverworld=0
Problem is actualy when i change this to not 0.

**Comment by Krysinski** [comment]
#2243 
Non-cargo rockets with plaer cant reach destination launch pad in multiverse dimentions. Cargo rockets do this normal. Also empty non-cargo rocket just disappeared when it was sent to multiverse dimention.
This tested with I:idDimensionOverworld=0 in a config.

**Comment by radfast** [comment]
Can anyone report whether this has been fixed by #2332?

**Comment by radfast** [comment]
> Problem is actualy when i change this to not 0.

The problem in the first post, the crash / kick from server, was because you had the idDimensionOverworld set to different values on the CLIENT and on the SERVER.  Code patch  42b51bb  fixes this for Galacticraft's own space stations.

**But**: you can maybe still have the same problem with space stations around other planets in the **add-on** packs, if the planet dimension IDs for those other planets are set differently in the configs on client and server .... this is not recommended!  @BlesseNtumble you may be interested?

------

> Non-cargo rockets with player cant reach destination launch pad in multiverse dimensions.

This is not connected with the Space Station creation issue.  Please open a new Issue report for this, if it is still a problem.
